snuba/environment.py

```diff
@@ -86,8 +86,15 @@ def setup_sentry() -> None:
         ],
         release=os.getenv("SNUBA_RELEASE"),
         traces_sample_rate=settings.SENTRY_TRACE_SAMPLE_RATE,
+        # "turn on" profiling without sending anything --
+        # nonzero value required for ondemand profiling as well
+        # profiles_sample_rate=0.00001,
```

Comment:
I believe this isn't necessary to turn on profiling. Your trick to set `transaction._profile.sampled = True` should suffice for it to start the profiler.

Reply:
without setting either a profiling_sampler or a nonzero sample rate, there will be no global scheduler, and so this assertion will fail: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/profiler.py

i could run my own scheduler and pass it in, but then i might have two schedulers when combined with the auto-instrumentation for flask that we also want to deploy at some point.